src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/ResourceFormStore.js

```diff
@@ -87,15 +87,114 @@ export default class ResourceFormStore extends AbstractFormStore implements Form
         });
     };
 
-    @action handleSchemaResponse = ([schema, jsonSchema]: [Schema, Object]) => {
+    handleSchemaResponse = ([schema, jsonSchema]: [Schema, Object]) => {
         this.validator = jsonSchema ? ajv.compile(jsonSchema) : undefined;
         this.pathsByTag = {};
 
-        this.schema = schema;
-        this.addMissingSchemaProperties();
-        this.setSchemaLoading(false);
+        return this.requestRemoteDataAndMerge(this.schema, schema).then(action(() => {
+            this.schema = schema;
+            this.addMissingSchemaProperties();
+            this.validate();
+            this.setSchemaLoading(false);
+        }));
     };
 
+    requestRemoteDataAndMerge = (localSchema: Schema, remoteSchema: Schema) => {
+        if (localSchema) {
+            return this.resourceStore.requestRemoteData({template: this.type}).then((data: Object) => {
+                const result = this.mergeData(localSchema, remoteSchema, this.data, data);
+                this.setMultiple(result);
+            });
+        }
+        return Promise.resolve();
+    };
+
+    mergeData(
+        localSchema: Schema,
+        remoteSchema: Schema,
+        localData: Object,
+        remoteData: Object
+    ) {
+        let result = {};
+        if (!localSchema || !remoteSchema) {
+            return result;
+        }
+
+        for (const name in remoteSchema) {
+            const {items: remoteItems, type: remoteType, types: remoteTypes} = remoteSchema[name];
+            const {items: localItems, type: localType, types: localTypes} = localSchema[name] || {};
+
+            if (remoteType === SECTION_TYPE && remoteItems &&
```

Comment:
think we need to split this into two condition blocks to handle a case where only one of the templates has a section 🙈